mlir/test/CAPI/CMakeLists.txt

```diff
@@ -21,34 +21,34 @@ target_link_libraries(mlir-mixr-capi-test
   MLIRCAPIRegisterRocMLIR
 )
 
-add_llvm_executable(mlir-tosa-miir-test
-  EXCLUDE_FROM_ALL
-  tosa_miir.cpp
-)
-llvm_update_compile_flags(mlir-tosa-miir-test)
+# add_llvm_executable(mlir-tosa-miir-test
```

Comment:
Also add a note to say these are archived

Reply:
Apologies, I missed that comment, I'll add the note.